windowsdoc.md

```diff
@@ -127,6 +128,15 @@ provide a custom timeout period using `--eval-timeout` argument. So if you face
 this error, the best option is to install Nodejs on your local system. If you
 can't then use the `--eval-timeout` argument and set a higher timeout value.
 
+### Fatal error occurs when executing a python command Line instruction as a job on python version 3.
+
+If your workflow creates a python command line instruction (like we do in this
+[unit test](https://github.com/common-workflow-language/cwltool/blob/master/tests/test_examples.py#L117) ) and you get a error `Fatal Python error: Failed to initialize Windows
```

Comment:
as per https://github.com/common-workflow-language/cwltool/pull/511/commits/d8d9b6647bf5dcdb89182c96842f989ccbf6dd5a this commit will be removed/reverted